### DISCUSSION TITLE
Get name of repository

### DISCUSSION THREAD

**Discussion by klieret** [lindell/multi-gitter#299] [Q&A] [answered]
I want to implement conditional logic based on the repository name (basically checking if something is a fork, for which I need to query the github API, which needs the repository name).

But how can I actually get the repository name from within a script that is being run by multi-gitter?

My first assumption was that `multi-gitter` would clone the repository in a directory of the same name (so I could simply get the current directory name), but that doesn't seem to be the case.

Any other way? Is there perhaps an environment variable could be set?

**Comment by lemeurherve** [comment, marked as answer]
The name of the repository is available via the `REPOSITORY` environment variable, as shown in the first example script of the README:
https://github.com/lindell/multi-gitter/blob/master/README.md?plain=1#L731-L732

```sh
#!/bin/bash
# This script should be used with the print command.
mkdir -p ~/multi-gitter/$REPOSITORY
cp -r . ~/multi-gitter/$REPOSITORY
```

**Reply by klieret** [reply]
Thanks for the speedy reply, didn't find the example when I was searching before!
